treeherder/perf/models.py

```diff
@@ -649,3 +649,11 @@ class PerformanceSettings(models.Model):
 
     class Meta:
         db_table = "performance_settings"
+
+
+class PerformanceAlertTag(models.Model):
```

Comment:
We're only using an `id` & a `name`. Might as well inherit from [NamedModel](https://github.com/mozilla/treeherder/blob/7c76cfb2b92addcf4e0fa7aab1ddadf52d343395/treeherder/model/models.py#L53).